bin/client.py

```diff
@@ -300,18 +300,18 @@ def main():
                             action='version',
```

Comment:
Need to remove the `(optional)` from the descriptions now it's deprecated.
Import ordering
Whitespace